src/lib/logging/types.ts

```
@@ -0,0 +1,20 @@
+export enum LogLevel {
+  DEBUG = 10,
```

Biome (2.3.13)

[error] This enum member name should be in PascalCase.



(lint/style/useNamingConvention)